src/main/java/com/service/CreatePersonService.java

```diff
@@ -154,4 +187,72 @@ public EStatus createPersonStatus() {
             }
         }
     }
+
+    public void readPropertyFileAndExecuteReadFileLinesMethod() throws IOException {
```

Comment:
I use your method and I dont care how you do it I just want to know what is the final result. Rename properly
Also don't throw generic exceptions, create one specific for your purpose and wrap the generic one in yours.For example if you have exception named for example **ReadPersonException** 
try {
// something throwing IOException
} catch(IOException e){
throw new ReadPersonException("some message",e);
}